press/templates/emails/suspended_sites_archival.html

```diff
@@ -13,6 +13,9 @@
 	    <br><br>
 	    In order to activate your account and ignore data loss on archival you can add prepaid credits to your account or settle the payment via card. Login to <a href="/dashboard/billing" style="text-decoration: underline;">Frappe Cloud Dashboard</a> for more details.
 	    For more payment options, you can refer to <a href="/payment-options" style="text-decoration: underline;">this link</a>.
+            <br>
+            We're sad to see you discontinue usage of Frappe Cloud and value your feedback. Please fill this <a href="frappecloud.com/frappe-cloud-feedback-form">form</a> or <a href="https://calendly.com/neha-frappe/frappe-user-interviews">block 15-min</a> with our team to share your experience.
+            If you have any concerns or questions, contact our support team at support.frappe.io.
```

Comment:
The template should be moved to UI side / customisable instead of residing in source code 😄

Reply:
Removed